tests/baselines/reference/arrayMapPredicates1.errors.txt

```diff
@@ -50,17 +48,29 @@ arrayMapPredicates1.ts(46,10): error TS2345: Argument of type 'ArrayPredOverload
     declare const asn: string[]|number[];
     asn.map(pisect1); // 5.2.2. error
             ~~~~~~~
-!!! error TS2345: Argument of type 'ArrayPredISect<string, number>' is not assignable to parameter of type '((value: string, index: number, array: string[]) => number) & ((value: number, index: number, array: number[]) => number)'.
-!!! error TS2345:   Type 'ArrayPredISect<string, number>' is not assignable to type '(value: string, index: number, array: string[]) => number'.
-!!! error TS2345:     Type 'string' is not assignable to type 'number'.
+!!! error TS2345: Argument of type 'ArrayPredISect<string, number>' is not assignable to parameter of type '(value: string | number, index: number, array: (string | number)[]) => number'.
+!!! error TS2345:   Types of parameters 'value' and 'value' are incompatible.
+!!! error TS2345:     Type 'string | number' is not assignable to type 'string'.
+!!! error TS2345:       Type 'number' is not assignable to type 'string'.
     asn.map(pisect2);
+            ~~~~~~~
+!!! error TS2345: Argument of type 'ArrayPredISect<unknown, unknown>' is not assignable to parameter of type '(value: string | number, index: number, array: (string | number)[]) => unknown'.

```

Comment:
This is one of two added errors.  Note that the previous version is an error whenever the a map function output is more specific than unknown.